src/snapshot/svgtoimg.js

```diff
@@ -18,20 +18,6 @@ function svgToImg(opts) {
         var Image = window.Image;
         var svg = opts.svg;
         var format = opts.format || 'png';
-
-        // IE only support svg
-        if(Lib.isIE() && format !== 'svg') {
```

Comment:
So this means that `Lib.isIE` return true in Edge? Is there an easy way to fix that? We could add an `Lib.isMSbrowser` if ever we need IE+Edge logic.

Reply:
I'd frankly vote to go for explicit feature detection - rename the current `Lib.isIE` to something like `usesMsSaveBlob`, and detect other stuff as needed like @arbscht's suggested `canExportFromCanvas`.

image export is currently the only place we use `Lib.isIE` anyway.

Reply:
Looks like `isIE` is only used in for image/download purposes:

![image](https://user-images.githubusercontent.com/6675409/31500285-0472c988-af35-11e7-9a12-cf08319d53e3.png)

so yeah, explicit detection would be best.